packages/x-data-grid/src/components/containers/GridRootStyles.ts

```diff
@@ -188,6 +181,16 @@ export const GridRootStyles = styled('div', {
     },
   };
 
+  const focusOutlineWidth = 1;
+  const columnSeparatorTargetSize = 10;
+  const columnSeparatorOffset = -5;
+
+  const separatorIconDragStyles = {
+    width: 3,
+    rx: 1.5,
+    x: 10.5,
+  };
```

Comment:
Can be moved outside of `GridRootStyles` since it's static

Reply:
Fixed https://github.com/mui/mui-x/pull/14293/commits/81a44dda2a2a4b596aaae8de531b2214fce0ae3e

Reply:
Same for the constants above that object 🙂

Reply:
🤦 fixed https://github.com/mui/mui-x/pull/14293/commits/d18c8855af6c221986984e5add9e2b89ee79e00b#diff-e1db53193a627d36d1cd3f651a954ee1e32ecb93caa020446631d722bb1be75cR41